recipes/wt/all/conanfile.py

```diff
@@ -203,6 +203,13 @@ def build(self):
         tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"), "INCLUDE(cmake/WtFindPostgresql.txt)", "#INCLUDE(cmake/WtFindPostgresql.txt)")
         if self.settings.os != "Windows":
             tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"), "INCLUDE(cmake/WtFindOdbc.txt)", "#INCLUDE(cmake/WtFindOdbc.txt)")
+
+
+        if tools.Version(self.version) == "4.5.0" and self.settings.compiler == "gcc" and tools.Version(self.settings.compiler.version) >= "11.0.0":
```

Comment:
We don't apply patches according version. Is it backward compatible? If yes, remove this condition and apply for any version.